tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/attachment_options.py

```diff
@@ -9,10 +9,9 @@
 
 class AttachmentOptions(Base):
```

Comment:
As we're using the activity selector from the system app, I'd delete this Page Class in order to use the [one from System](https://github.com/steveck-chung/gaia/blob/new-message-attachment-menu/tests/python/gaia-ui-tests/gaiatest/apps/system/regions/activities.py). And add the locators and methods there, if needed.